system_table.py

```diff
@@ -54,9 +57,9 @@ def propagate_variable_uses(bv: BinaryView, func: Function, var: SSAVariable, fu
             if not isinstance(target, Constant):
                 continue
 
-            type_name = get_type_name(var.type.target)
-            log_info(f"Propagating {type_name} pointer to data variable at {hex(target.constant)}")
-            bv.define_user_data_var(target.constant, var.type, types_to_propagate[type_name])
+            type_name = get_type_name(var.type)
+            bv.define_user_data_var(target.constant, var.type,
```

Comment:
Recommend `types_to_propagate.get(type_name)` for the third param (which returns `None` if the key `type_name` is not in the dict)